test-packages/integration-tests/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../tsconfig.json",
```

Comment:
Is this needed? Or is it just a left-over from trying things out?

Reply:
it is necessary. It turns out that ts-jest needs a tsconfig.json implcitly.